packages/datadog-plugin-aerospike/test/index.spec.js

```diff
@@ -0,0 +1,300 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const { ERROR_MESSAGE, ERROR_TYPE, ERROR_STACK } = require('../../dd-trace/src/constants')
+
+describe('Plugin', () => {
+  let client
+  let aerospike
+  let config
+  let tracer
+  let ns
+  let set
+  let userKey
+  let key
+  let keyString
+
+  describe('aerospike', () => {
+    withVersions('aerospike', 'aerospike', version => {
+      beforeEach(() => {
+        tracer = require('../../dd-trace')
+        aerospike = require(`../../../versions/aerospike@${version}`).get()
+      })
+
+      beforeEach(() => {
+        ns = 'test'
+        set = 'demo'
+        userKey = 'key'
+
+        config = {
+          hosts: '127.0.0.1:3000',
+          port: '3000'
+        }
+
+        client = aerospike.client(config)
+        key = new aerospike.Key(ns, set, userKey)
+        keyString = `${ns}:${set}:${userKey}`
+      })
+
+      after(() => {
+        return agent.close({ ritmReset: false })
+      })
+
+      describe('without configuration', () => {
+        before(() => {
+          return agent.load('aerospike')
+        })
+
+        afterEach(() => {
+          client.close()
+        })
+
+        describe('client', () => {
+          it('should instrument put', done => {
+            agent
+              .use(traces => {
+                const span = traces[0][0]
+                const resource = `Put`
```

Comment:
This is only used in one place. Could just embed it in-place. But also, it shouldn't use backticks if it's a static string like this.